streams/examples/src/main/java/org/apache/kafka/streams/examples/pageview/PageViewTypedDemo.java

```diff
@@ -195,8 +260,48 @@ public void run() {
 
         try {
             streams.start();
+            // very simple data generators for demonstration purposes.
```

Comment:
Is this necessary? We do not include sample data production / consumption intentionally across all examples because we document in the tutorials to start these examples as pure streams app only.

Reply:
I see. I wasn't sure. I'll take them back out, but I do think we should document the input format, so I'll do that.